### PR TITLE
Quiz instruction page

### PR DESCRIPTION
**Pull request by Ankur9669**
image

### CODE REVIEW

File: common.css [hunk 1]
```diff
@@ -84,11 +87,47 @@ span{
 }
 .page-container{
     display: grid;
-    grid-template-columns: 300px auto;
+    grid-template-columns: 18.75rem auto;
+}
+.quiz-heading-container{
+    display: flex;
+    justify-content: space-between;
+    margin: 2rem 2rem;
```
Comment (sri-rishi) [inline common.css:95]: use shorthand property

File: quizinstruction.css [hunk 2]
```diff
@@ -0,0 +1,35 @@
+.quiz-info-item{
+    display: flex;
+    justify-content: space-around;
+}
+.quiz-info{
+    margin-left: 10%;
+    margin-right: 1rem;
+}
+.quiz-info-container .image-container{
+    margin: 0rem 2rem;
```
Comment (sri-rishi) [inline quizinstruction.css]: use 0 instead of 0rem

File: quizinstruction.css [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    max-height: 300px;
+    max-width: 400px;
```
Comment (sri-rishi) [inline quizinstruction.css:12]: use rem here

**Review by sri-rishi — CHANGES_REQUESTED** [review]

**Review by yaswanthmyneni — APPROVED** [review]
The code looks good to me. UI looks great.

File: quizinstruction.css (hunk 3 above)
Comment (Ankur9669) [reply quizinstruction.css:12]: Actually i don't want it to change according to font size that is why i have used px

File: quizinstruction.css (hunk 2 above)
Comment (Ankur9669) [reply quizinstruction.css]: Done

**Review by sri-rishi — APPROVED** [review]

**Pull request merged**